docs-ref-conceptual/eclipse/java-sdk-azure-get-started-eclipse.md

```diff
@@ -0,0 +1,419 @@
+  "displayName": "AzureJavaTest",
+  "name": "http://AzureJavaTest",
+  "password": password,
+  "tenant": "tttttttt-tttt-tttt-tttt-tttttttttttt"
+}
+```
+
+Next, copy the following into a text file on your system:
+
+```text
+# sample management library properties file
+subscription=ssssssss-ssss-ssss-ssss-ssssssssssss
+client=cccccccc-cccc-cccc-cccc-cccccccccccc
+key=kkkkkkkkkkkkkkkk
+tenant=tttttttt-tttt-tttt-tttt-tttttttttttt
+managementURI=https\://management.core.windows.net/
+baseURL=https\://management.azure.com/
+authURL=https\://login.windows.net/
+graphURL=https\://graph.windows.net/
+```
+
+Replace the top four values with the following:
+
+- subscription: use the *id* value from `az account show` in the Azure CLI 2.0.
+- client: use the *appId* value from the output taken from a service principal output.
+- key: use the *password* value from the service principal output.
+- tenant: use the *tenant* value from the service principal output.
+
+Save this file in a secure location on your system where your code can read it. You may use this file for future code so it's recommended to store it somewhere external to the application in this article.
+
+Set an environment variable `AZURE_AUTH_LOCATION` with the full path to the authentication file in your shell.
+
+```bash
+export AZURE_AUTH_LOCATION=/Users/raisa/azureauth.properties
+```
+
+If you're working in a windows environment, add the variable to your system properties. Open PowerShell and, after replacing the second variable with the path to your file, enter the following command:
+
+```powershell
+[Environment]::SetEnvironmentVariable("AZURE_AUTH_LOCATION", "C:\<fullpath>\azureauth.properties", "Machine")
+```
+
+## Create a new Maven project
+
+> [!NOTE]
+> This guide uses Maven build tool to build and run the sample code, but other build tools such as Gradle also work with the Azure libraries for Java. 
+
+Open Eclipse, select **File** -> **New**. In the new window that appears open the Maven folder then select Maven Project. 
+
+Leave the selections on the next screen defaults, and select **Next**. Do the same for this screen regarding archetypes.
+
+When you come to the screen asking for groupID, ArtifactID, etc. Enter "com.fabrikam" for the groupID and enter "AzureApp" for the artifactID.
+
+Now, open the pom.xml file. Inside the `dependencies` tag add the following code:
+
+```XML
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure</artifactId>
+    <version>1.3.0</version>
+</dependency>
+<dependency>
+    <groupId>com.microsoft.azure</groupId>
+    <artifactId>azure-storage</artifactId>
+    <version>5.0.0</version>
+</dependency>
+<dependency>
+    <groupId>com.microsoft.sqlserver</groupId>
+    <artifactId>mssql-jdbc</artifactId>
+    <version>6.2.1.jre8</version>
+</dependency>
+```
+
+Now, save the pom.xml. This prompts Eclipse to download all the specified dependencies. This may take a moment.
+   
+## Install the azure toolkit for Eclipse
```

Comment:
Do we need to have people install the toolkit to complete this quickstart? It's not required and it's a hurdle to successful completion.

Reply:
Thanks for the review! I'll incorporate all your other feedback but this one I would prefer to keep. If only so that we have articles which give the user a complete end to end dev environment. Even though I realize they don't use the toolkit anywhere in this article. My goal is that the user leaves this article with an environment ready for any type of development.

My rationale: 
Most (if not all) other languages which are featured on azure docs achieve this easily. Installing their packages or configuring their environment is usually a single step which gives them an environment which can interface with azure both programmatically and for deploying web apps. Java, which doesn't inherently do that, should be synchronous with that. Also it's a pretty trivial step.